Useful tools & utilities/Figma.md

```diff
@@ -0,0 +1,124 @@
+access modes entail, check out the following [link](https://help.figma.com/hc/en-us/articles/360039970673-Team-permissions).
+
+## Figma essentials
+
+Let's take a look at some of the things which you'll probably be using very frequently as a developer.
+
+### Frames
+
+![Figma Frames](/img/figma/figma_frames.png "Frames in Figma")
+
+Frames serve as top-level containers for most elements in Figma. Usually, each screen has a separate frame and they are all listed
+on the left side as shown in the image above. In this example project, the frames are located in the Layers tab, however, keep in
+mind that this is customizable and the tab naming in your project might differ from this project's conventions (for example, you might have
+a Pages tab rather than a Layers tab). What's important is to know the fundamentals and what Figma provides as functionalities and then,
+minor things like naming conventions and way of structuring stuff within the team is easily manageable.
+
+Each frame can be expanded and its sub-frames and elements will be shown as in the Register frame on the image above. You can click around
+and explore what each frame consists of. You can select a particular frame or an element from the frame directly on the design canvas as
+well.
+
+### Element selection, code inspection, layout measurements and constraints
+
+If you want to get more details about a certain element/component/part of the design, you can select it and then check out its details on the
+right side of the Figma file.
+
+Here's an example when the "NEXT" button from the Register screen is selected:
+
+![Figma Element Inspection](/img/figma/figma_button_inspection.png "Element inspection in Figma")
+
+In the Inspect tab on the right side, there is a Properties section which has some basic information about the dimensions of the selected
+element. In the Code section, there is a small code snippet which can be used as a guideline for implementing the design in code. On this
+image, Android is selected as the preferred platform in the drop-down menu, however, you can choose iOS or CSS and the code snippets will be
+displayed accordingly. Depending on the selected element, more sections might be present in this Inspect tab, such as sections with
+information about Colors, Borders, Typography, etc. If you hover around the designs and have the button selected, you can see some lines
+which represent its constraints. You can also view the margins, spacing and other dimensions by hovering over particular regions in the
+designs.
+
+### Exports
+
+In the Figma files for the project you'll be working on, there will probably be an Export tab on the right side as well when a certain
+element/component is selected. If you have edit access, you'll have the Export section in the Design tab, however, for developers, it is
+usually enough to have only view access to the Figma files.
```

Comment:
Maybe we should drop this part: _however, for developers, it is
usually enough to have only view access to the Figma files._
Let's not assume. 
Also, I am pretty sure that I don't have edit access on the project I am working on but I have an export section?
Can you fact-check this part?

Reply:
Yes, what I wanted to emphasise in this part is that the Export section is located at a different place depending on the access rights (either in the Design tab as a section or it has its own Export tab - what David also mentioned). I used the terms “section” and “tab” in my explanation to distinguish where and in which form the export options are available, but if it is still a bit ambiguous and you believe that this positioning varies a lot from version to version, I’ll do some rephrasing. Also, I’ll omit the assumption about the developer’s access rights if you think that it also adds to the confusion.